src/optimization/hooke_jeeves_method.cpp

```diff
@@ -0,0 +1,149 @@
+#include <k52/optimization/hooke_jeeves_method.h>
+
+#ifdef BUILD_WITH_MPI
+
+#include <boost/mpi.hpp>
+#include <k52/parallel/mpi/constants.h>
+
+#endif
+
+#include <iostream>
+#include <cmath>
+#include <algorithm>
+#include <stdexcept>
+
+#include <k52/optimization/params/i_continuous_parameters.h>
+
+using ::std::vector;
+
+namespace
+{
+	double kAcceleration = 3.5;
```

Comment:
Yep, researches, that was in our Methods of computational experiment course, show that minimum number of iterations will be achieved by coefficient that equals 3.5.